src/rx-query.ts

```diff
@@ -816,7 +828,12 @@ async function __ensureEqual<RxDocType>(rxQuery: RxQueryBase<RxDocType, any>): P
                 }
             }
 
-            if (rxQuery.op === 'count') {
+            if (rxQuery.includesDeleted) {
```

Comment:
I can't quite remember, but do we need to specifically handle `count` operations for a selector that contains a true-like value for `_deleted`, or is this automatically covered by the slow count option?

Reply:
Pseudo code

```ts
} else if (rxQuery.op === 'count' && rxQuery.includesDeleted) {
   rxQuery._execOverDatabase().then((newResultData) => {
    rxQuery._setResultData(newResultData ? newResultData.length : 0);
    return true;
  });
}
```

Reply:
This is a good point that I have not tested yet. Could I verify this by using the field that Tristan mentioned [here](https://github.com/readwiseio/rekindled/pull/4747/files/f88853c71603bfa2a559e816e4a0200d894d75c9#r1805719085) `'rxdbOnly.indexFields.triage_status_exists'`  and then check the correct count?